handlers/main.yml

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+---
+# Put handlers for tasks here.
+
+# Example:
+- name: template handler to restart services
+  service:
+    name: "{{ item }}"
+    state: restarted
+  loop: "{{ __template_services }}"
```

Comment:
remove the `# Example:` line
change the `name` to be `certificate handler` instead of `template handler`
use `__certificate_services` instead of `__template_services`

Reply:
I've removed handlers https://github.com/linux-system-roles/certificate/pull/1/commits/7012d594bd5ccbbcb7b03ce7bf0922f2ac9425cd